jtlibrary/python/jtmodules/src/jtmodules/separate_clumps.py

```diff
@@ -77,9 +83,25 @@ def main(mask, intensity_image, min_area, max_area,
 
     if plot:
         from jtlib import plotting
+        cut_mask = (mask > 0) - (separated_mask > 0)
```

Comment:
I'm not sure if this is what you want: `cut_mask` will be `True` where *either* `mask` *or* `separated_mask` are positive.

If you wanted to have the bits where `mask` is on but `separated_mask` is off, then that's `(mask > 0) & ~(separated_mask > 0)`, or equivalently `(mask > 0) & (separated_mask == 0)`.

The expressions evaluate to the same result if and only if `separated_mask > 0` is a subset of `mask > 0`.

Reply:
Solved in #162